app/App.js

```diff
@@ -15,11 +16,11 @@ import { ViroVRSceneNavigator, ViroARSceneNavigator } from 'react-viro';
 var sharedProps = {
   apiKey: 'API_KEY_HERE',
 };
-
+// CR * var/pasted code
```

Comment:
Lots of use of `var` around these parts, which indicates to me that this is copy-pasted example code, or auto-generated code (possibly from `react-viro-cli`). This is not an immediate priority, but you want to make sure the code in your project matches the way *you* would write it.